Garima/AuthSession/auth_session/auth_session/Controllers/loginController.cs

```diff
@@ -0,0 +1,257 @@
+                catch (Exception ex)
+                {
+                    return Unauthorized();
+                }
+            }
+            catch (Exception ex)
+            {
+
+            }
+            return BadRequest();
+        }
+
+
+
+        //PUT: api/login/5
+        [HttpPut("{id}")]
+        public void Put(int id, [FromBody]string value)
+        {
+        }
+
+        //DELETE: api/ApiWithActions/5
+        [HttpDelete("{id}")]
+        public void Delete(int id)
+        {
+        }
+
+
+        //auth_sessionContext db = new auth_sessionContext();
+        //[Route("login")]
+        //[HttpPost]
+
+        //public IActionResult Signup([FromBody]dynamic loginData)
+        //{
+        //    try
+        //    {
+        //        StringValues _username;
+        //        StringValues _password;
+        //        Request.Headers.TryGetValue("email", out _username);
+        //        Request.Headers.TryGetValue("password", out _password);
+
+        //        String username = _username.FirstOrDefault();
+        //        String password = _password.FirstOrDefault();
+
+        //        Signup loggedinUser = db.Signup.Find(username);
+        //        try
+        //        {
+        //            if (BCrypt.Net.BCrypt.Verify(password, loggedinUser.Password))
+        //            {
+        //                String tokenString = GenerateJSONWebToken(loggedinUser);
+        //                return Ok(new { token = tokenString });
+        //            }
+        //        }
+        //        catch (Exception ex)
+        //        {
+        //            return Unauthorized();
+        //        }
+        //    }
+        //    catch (Exception ex)
+        //    {
+
+        //    }
+        //    return BadRequest();
+        //}
+
+
+        //Signup
+        [Route("signup")]
+            [HttpPost]
+            public IActionResult Singup([FromBody]Signup signupUser)
+            {
+                try
+                {
+                    signupUser.Password = BCrypt.Net.BCrypt.HashPassword(signupUser.Password);
+                    db.Signup.Add(signupUser);
+                    db.SaveChanges();
+                    return Ok();
+
+                }
+                catch (Exception ex)
+                {
+
+                }
+                return BadRequest();
+            }
+
+            [Route("sample")]
+            [HttpGet]
+            [Authorize]
+            public IActionResult sampleAuthRoute()
+            {
+                try
+                {
+                    var currentUser = HttpContext.User;
+                    //TODO: Make claims work, currently not working
+                    //if (currentUser.HasClaim(c => c.Type == "Email"))
+                    //{
+                    //    String email = currentUser.Claims.FirstOrDefault(c => c.Type == "Email").Value;
+                    //}
+                    return Ok(new { message = "Sample page working" });
```

Comment:
You just copied and pasted my code. Did not even remove the comments. The code that I shared was meant for reference not for copying